release_notes.md

```diff
@@ -4,6 +4,7 @@
 -->
 - Fixed [bug](https://github.com/Azure/azure-functions-durable-extension/issues/1467) in sync triggers operations for Durable Functions using custom storage account connection strings.
 - Update PowerShell Worker to 3.0.557 (PS6) [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.557) and 3.0.560 (PS7) [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v3.0.560)
+- Update Microsoft.Azure.WebJobs and Microsoft.Azure.WebJobs.Logging.ApplicationInsights to v3.0.25 [Release Notes](https://github.com/Azure/azure-webjobs-sdk/releases/tag/v3.0.25)
```

Comment:
Should we add a note that we are updating http ext?

Reply:
yeah let me add the note